docs/staging.adoc

```diff
@@ -0,0 +1,67 @@
+= Staging your application with the OpenShift registry
+
+You can prepare your application for production locally with the internal OpenShift
+Docker registry before you push your containerized application to production on OpenShift.
+
+The scenario assumes the following parameters:
+
+- OpenShift registry endpoint is *hub.openshift.adb*
+- OpenShift instance URL is *https://10.1.2.2:8443*
+- Username/password credentials is *openshift-dev/devel*
+- Default ADB project name is *sample-project*
+- Application name is *my-app*
+
+This scenario is intended to illustrate how to test an application that isn't already
+pushed to a public instance of OpenShift and that is not accessible by OpenShift
+`build config`.
+
```

Comment:
@thatdocslady This point about `build config` is not clear to me.  What are we trying to communicate here?